module/userProjects/helper.js

```diff
@@ -4120,6 +4146,79 @@ function _projectData(data) {
 	})
 }
 
+/**
+ * Fetch program entity data.
+ * @method
+ * @name __getProgramEntityData
+ * @param {String} programId - program id.
+ * @returns {Object} program entity related data.
+ */
+function __getProgramEntityData(programId) {
+	return new Promise(async (resolve, reject) => {
+		try {
+			// Fetch program details
+			let programDetails = await programQueries.programsDocument(
+				{
+					_id: ObjectId(programId),
+				},
+				['scope']
+			)
+			if (!(programDetails.length > 0)) {
+				throw {
+					status: HTTP_STATUS_CODE.bad_request.status,
+					message: CONSTANTS.apiResponses.PROGRAM_NOT_FOUND,
+				}
+			}
+			programDetails = programDetails[0]
+			let entityTypes = programDetails.scope.entityType.split(',')
```

Comment:
@Prajwal17Tunerlabs What ever type of entities we store in scope all their type are mentioned in entityType?

Reply:
yes @VISHNUDAS-tunerlabs